Tribler/Main/vwxGUI/GuiUtility.py

```diff
@@ -112,6 +131,7 @@ def ShowPlayer(self):
 
     @forceWxThread
     def ShowPage(self, page, *args):
+        assert wx.Thread_IsMain()
```

Comment:
Remove all these redundant asserts, if it's decorated with `forceWxThread` it WILL be executed on the wx thread.